pyramid_oereb/contrib/data_sources/create_tables.py

```diff
@@ -241,11 +241,20 @@ def create_theme_tables():
         default=False,
         help='Is the yaml file using a c2ctemplate style (starting with vars)'
     )
+    parser.add_option(
+        '--config-source',
```

Comment:
If I use
`--config-source pyramid_oereb.contrib.sources.plr_oereblex.DatabaseOEREBlexSource`
sql statements for creating the tables related to the standard data source are generated for the OerebLex topic.

Is it also possible to generate sql statements for creating tables for OerebLex? If not, maybe that should be stated clearer.

Reply:
> Is it also possible to generate sql statements for creating tables for OerebLex? If not, maybe that should be stated clearer.

Yes this should work, if you specify this in your config.yaml and then use this config-source parameter. The code that will be run is the same as the one which is run when the automated test suite is run, and, as I understand, it works fine there. This is why I added this new option to this script, because we get this possibility "for free".

Reply:
In script 
https://github.com/openoereb/pyramid_oereb/blob/master/pyramid_oereb/contrib/data_sources/oereblex/create_tables.py
the creation of tables is based on
https://github.com/openoereb/pyramid_oereb/blob/fa125c299c46467756f0a21d87cbad094e71ee93/pyramid_oereb/contrib/data_sources/oereblex/create_tables.py#L5

As far as I see, it is always the standard table structure that is created. Or do I miss something?

Reply:
Yes I think you are right, I misunderstood and I have now found a "create_oereblex_tables" script.
So the new option does not make sense.
I will simplify this PR accordingly.